### DISCUSSION TITLE
window.scrollTo not working

### DISCUSSION THREAD

**Discussion by islamzaoui** [sveltejs/kit#10698] [Q&A]
I have this function that I use in Onmount () to scroll back to the top when navigating between pages
```
        function scrollToTop() {
		window.scrollTo({ top: 0, left: 0, behavior: 'smooth' })
	}
```
but for some reason, it's not working
or give me alternative solutions

**Comment by Angus-Paillaugue** [comment]
Hi, I tested you code and it's working fine. Is there an error thrown or is it just not scrolling to the top ?
Btw, here is the code I tested : 

```
import { onMount } from 'svelte';

onMount(() => {
    scrollToTop();
});
function scrollToTop() {
    window.scrollTo({ top: 0, left: 0, behavior: 'smooth' });
}
```

**Reply by ben-arslan** [reply]
voici le code complet

**Comment by ben-arslan** [comment]
<script>
        function scrollToTop() {
            window.scrollTo({ top: 0, behavior: 'smooth' });
        }

        function scrollToBottom() {
            window.scrollTo({ top: document.body.scrollHeight, behavior: 'smooth' });
        }
    </script>
